screen/model/media/mediahandler.js

```diff
@@ -47,6 +55,22 @@ module.exports = class MediaHandler {
         }
     }
 
+    setViewport(x, y, width, height, percentage = false){
+        let suffix = (percentage ? "%" : "px");
```

Comment:
const suffix? Since it doesn't change.

I haven't paid attention to this in other reviews, so I might have missed some similar cases.